Auto update failing for vulkan-validation-layers

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [vulkan-validation-layers](https://github.com/termux/termux-packages/blob/9cdae6f053a9cf4d25df7fd2b8e3843d1a405816/packages/vulkan-validation-layers) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating vulkan-validation-layers [Current version: 1.4.311]
INFO: package being updated to 1.4.312.
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.4.312.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 5640k    0 5640k    0     0  5310k      0 --:--:--  0:00:01 --:--:-- 5310k
100 6148k    0 6148k    0     0  5565k      0 --:--:--  0:00:01 --:--:-- 11.8M

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-validation-layers/build.sh b/packages/vulkan-validation-layers/build.sh
index f0212d4282..f6ffad6bbe 100644
--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.4.311"
+TERMUX_PKG_VERSION="1.4.312"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=d784393c9fd2b8c3f9ca71aa14cee5fd0690227e433c16895ea9c6db0e1b43a1
+TERMUX_PKG_SHA256=e761c55eb460cd8923f9c9772031e9e2b7ce215d01d0ebb6ca7e41f112df4faf
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
--------------------

bump(main/vulkan-validation-layers): 1.4.312


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-validation-layers exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-validation-layers for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  81692      0 --:--:-- --:--:-- --:--:-- 82081
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   4646      0 --:--:-- --:--:-- --:--:--  4653
gpg: Signature made Tue 08 Apr 2025 12:25:06 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  1 1673k    1 32433    0     0  64031      0  0:00:26 --:--:--  0:00:26 63970
100 1673k  100 1673k    0     0  1558k      0  0:00:01  0:00:01 --:--:-- 1559k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  78897      0 --:--:-- --:--:-- --:--:-- 79279
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   4876      0 --:--:-- --:--:-- --:--:--  4871
gpg: Signature made Sun 06 Apr 2025 06:21:20 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  75948      0 --:--:-- --:--:-- --:--:-- 76273
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5000      0 --:--:-- --:--:-- --:--:--  5018
gpg: Signature made Mon 07 Apr 2025 12:32:17 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.7.1 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.7 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libffi/libffi_3.4.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30588  100 30588    0     0   442k      0 --:--:-- --:--:-- --:--:--  445k
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.18 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.18 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.8.1 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.8.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  187k  100  187k    0     0  2027k      0 --:--:-- --:--:-- --:--:-- 2034k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.8.1 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.8.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 71040  100 71040    0     0   929k      0 --:--:-- --:--:-- --:--:--  937k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gdbm/gdbm_1.24_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  139k  100  139k    0     0  1974k      0 --:--:-- --:--:-- --:--:-- 1998k
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-posix-semaphore/libandroid-posix-semaphore_0.1-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4128  100  4128    0     0  66219      0 --:--:-- --:--:-- --:--:-- 66580
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2025.02.25 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2025.02.25 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.4.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.4.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.49.1-1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.49.1-1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.49.1-1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.47 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.16.1 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.16.1 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.2 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.12 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-1 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-1 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.6.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.12 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.12 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.12.9 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python/python_3.12.9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7417k  100 7417k    0     0  68.0M      0 --:--:-- --:--:-- --:--:-- 68.3M
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.12.9 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-tkinter/python-tkinter_3.12.9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1191k  100 1191k    0     0  12.9M      0 --:--:-- --:--:-- --:--:-- 13.0M
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.12.9 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-ensurepip-wheels/python-ensurepip-wheels_3.12.9_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1740k  100 1740k    0     0  17.3M      0 --:--:-- --:--:-- --:--:-- 17.3M
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.13.7 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2/libxml2_2.13.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  456k  100  456k    0     0  5076k      0 --:--:-- --:--:-- --:--:-- 5126k
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.13.7 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-utils/libxml2-utils_2.13.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30880  100 30880    0     0   436k      0 --:--:-- --:--:-- --:--:--  443k
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.13.7 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python/libxml2-python_2.13.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  188k  100  188k    0     0  2720k      0 --:--:-- --:--:-- --:--:-- 2738k
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.13.7 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python-static/libxml2-python-static_2.13.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 73752  100 73752    0     0  1027k      0 --:--:-- --:--:-- --:--:-- 1028k
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.23.1-1 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland/libwayland_1.23.1-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  119k  100  119k    0     0  1400k      0 --:--:-- --:--:-- --:--:-- 1409k
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.23.1-1 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-cross-scanner/libwayland-cross-scanner_1.23.1-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 84760  100 84760    0     0  1304k      0 --:--:-- --:--:-- --:--:-- 1313k
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-all...
Downloading dependency libxrandr@1.5.4 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrandr/libxrandr_1.5.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 21824  100 21824    0     0   288k      0 --:--:-- --:--:-- --:--:--  288k
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency spirv-headers@1.4.309.0 if necessary...
Found spirv-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-headers/spirv-headers_1.4.309.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  143k  100  143k    0     0   213k      0 --:--:-- --:--:-- --:--:--  213k
extracting spirv-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency spirv-tools@1.4.309.0 if necessary...
Found spirv-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-tools/spirv-tools_1.4.309.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1810k  100 1810k    0     0  4460k      0 --:--:-- --:--:-- --:--:-- 4459k
extracting spirv-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-headers@1.4.312 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-headers/vulkan-headers_1.4.312_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1521k  100 1521k    0     0  14.1M      0 --:--:-- --:--:-- --:--:-- 14.2M
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-utility-libraries@1.4.312 if necessary...
Found vulkan-utility-libraries in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-utility-libraries/vulkan-utility-libraries_1.4.312_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  514k  100  514k    0     0   565k      0 --:--:-- --:--:-- --:--:--  565k
extracting vulkan-utility-libraries to /home/builder/.termux-build/_cache-aarch64...
Downloading vulkan-validation-layers@1.4.312 source from 'https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.4.312.tar.gz' if necessary...
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.4.312.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 5632k    0 5632k    0     0  5618k      0 --:--:--  0:00:01 --:--:-- 5618k
100 6148k    0 6148k    0     0  5708k      0 --:--:--  0:00:01 --:--:-- 6981k
Applying patch: 0001-no-android.patch
Applying patch: 0002-getenv.patch
Applying patch: 0003-layers-utils-android_ndk_types.h.patch
Applying patch: 0004-setenv.patch
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The CXX compiler identification is Clang 18.0.3
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using parallel_hashmap as custom hash maps
-- Configuring done (0.3s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_C_FLAGS
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Generating done (0.0s)
-- Build files have been written to: /home/builder/.termux-build/vulkan-validation-layers/build
[1/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/error_location_helper.cpp.o
[2/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/error_location.cpp.o
[3/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/pnext_chain_extraction.cpp.o
[4/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/feature_requirements_helper.cpp.o
[5/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_object_types.cpp.o
[6/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/logging.cpp.o
[7/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/convert_utils.cpp.o
[8/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/hash_util.cpp.o
[9/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/image_layout_utils.cpp.o
[10/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_extension_utils.cpp.o
[11/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_extension_helper.cpp.o
[12/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/ray_tracing_utils.cpp.o
[13/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/text_utils.cpp.o
[14/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_dispatch_table_helper.cpp.o
[15/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_utils.cpp.o
[16/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vk_layer_config.cpp:46:13: warning: unused function 'PropCallback' [-Wunused-function]
   46 | static void PropCallback(void *cookie, [[maybe_unused]] const char *name, const char *value, [[maybe_unused]] uint32_t serial) {
      |             ^~~~~~~~~~~~
1 warning generated.
[17/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_struct_compare.cpp.o
[18/198] Building CXX object layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o
FAILED: layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DUSE_CUSTOM_HASH_MAP -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/external -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o -MF layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o.d -o layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_indexing_oob_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_indexing_oob_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_indexing_oob_pass.cpp:18:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/module.h:23:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/type_manager.h:22:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:417:19: error: no member named 'OpTaskSequenceCreateINTEL' in namespace 'spv'
  417 |         case spv::OpTaskSequenceCreateINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:418:19: error: no member named 'OpTaskSequenceGetINTEL' in namespace 'spv'
  418 |         case spv::OpTaskSequenceGetINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:420:19: error: no member named 'OpBitwiseFunctionINTEL' in namespace 'spv'
  420 |         case spv::OpBitwiseFunctionINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:429:19: error: no member named 'OpRoundFToTF32INTEL' in namespace 'spv'
  429 |         case spv::OpRoundFToTF32INTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:849:19: error: no member named 'OpTaskSequenceCreateINTEL' in namespace 'spv'
  849 |         case spv::OpTaskSequenceCreateINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:850:19: error: no member named 'OpTaskSequenceGetINTEL' in namespace 'spv'
  850 |         case spv::OpTaskSequenceGetINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:851:19: error: no member named 'OpTypeTaskSequenceINTEL' in namespace 'spv'
  851 |         case spv::OpTypeTaskSequenceINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:853:19: error: no member named 'OpBitwiseFunctionINTEL' in namespace 'spv'
  853 |         case spv::OpBitwiseFunctionINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:862:19: error: no member named 'OpRoundFToTF32INTEL' in namespace 'spv'
  862 |         case spv::OpRoundFToTF32INTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:1309:19: error: no member named 'OpTypeTaskSequenceINTEL' in namespace 'spv'
 1309 |         case spv::OpTypeTaskSequenceINTEL:
      |              ~~~~~^
10 errors generated.
[19/198] Building CXX object layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o
FAILED: layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DUSE_CUSTOM_HASH_MAP -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/external -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o -MF layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o.d -o layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_texel_buffer_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_texel_buffer_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_texel_buffer_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/module.h:23:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/type_manager.h:22:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:417:19: error: no member named 'OpTaskSequenceCreateINTEL' in namespace 'spv'
  417 |         case spv::OpTaskSequenceCreateINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:418:19: error: no member named 'OpTaskSequenceGetINTEL' in namespace 'spv'
  418 |         case spv::OpTaskSequenceGetINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:420:19: error: no member named 'OpBitwiseFunctionINTEL' in namespace 'spv'
  420 |         case spv::OpBitwiseFunctionINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:429:19: error: no member named 'OpRoundFToTF32INTEL' in namespace 'spv'
  429 |         case spv::OpRoundFToTF32INTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:849:19: error: no member named 'OpTaskSequenceCreateINTEL' in namespace 'spv'
  849 |         case spv::OpTaskSequenceCreateINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:850:19: error: no member named 'OpTaskSequenceGetINTEL' in namespace 'spv'
  850 |         case spv::OpTaskSequenceGetINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:851:19: error: no member named 'OpTypeTaskSequenceINTEL' in namespace 'spv'
  851 |         case spv::OpTypeTaskSequenceINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:853:19: error: no member named 'OpBitwiseFunctionINTEL' in namespace 'spv'
  853 |         case spv::OpBitwiseFunctionINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:862:19: error: no member named 'OpRoundFToTF32INTEL' in namespace 'spv'
  862 |         case spv::OpRoundFToTF32INTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:1309:19: error: no member named 'OpTypeTaskSequenceINTEL' in namespace 'spv'
 1309 |         case spv::OpTypeTaskSequenceINTEL:
      |              ~~~~~^
10 errors generated.
[20/198] Building CXX object layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o
FAILED: layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27c-api-24-v1/sysroot -DUSE_CUSTOM_HASH_MAP -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/external -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -isystem/data/data/com.termux/files/usr/include/c++/v1 -isystem/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o -MF layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o.d -o layers/gpuav/spirv/CMakeFiles/gpu_av_spirv.dir/descriptor_class_general_buffer_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_general_buffer_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpuav/spirv/descriptor_class_general_buffer_pass.cpp:17:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:417:19: error: no member named 'OpTaskSequenceCreateINTEL' in namespace 'spv'
  417 |         case spv::OpTaskSequenceCreateINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:418:19: error: no member named 'OpTaskSequenceGetINTEL' in namespace 'spv'
  418 |         case spv::OpTaskSequenceGetINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:420:19: error: no member named 'OpBitwiseFunctionINTEL' in namespace 'spv'
  420 |         case spv::OpBitwiseFunctionINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:429:19: error: no member named 'OpRoundFToTF32INTEL' in namespace 'spv'
  429 |         case spv::OpRoundFToTF32INTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:849:19: error: no member named 'OpTaskSequenceCreateINTEL' in namespace 'spv'
  849 |         case spv::OpTaskSequenceCreateINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:850:19: error: no member named 'OpTaskSequenceGetINTEL' in namespace 'spv'
  850 |         case spv::OpTaskSequenceGetINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:851:19: error: no member named 'OpTypeTaskSequenceINTEL' in namespace 'spv'
  851 |         case spv::OpTypeTaskSequenceINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:853:19: error: no member named 'OpBitwiseFunctionINTEL' in namespace 'spv'
  853 |         case spv::OpBitwiseFunctionINTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:862:19: error: no member named 'OpRoundFToTF32INTEL' in namespace 'spv'
  862 |         case spv::OpRoundFToTF32INTEL:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:1309:19: error: no member named 'OpTypeTaskSequenceINTEL' in namespace 'spv'
 1309 |         case spv::OpTypeTaskSequenceINTEL:
      |              ~~~~~^
10 errors generated.
[21/198] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_function_pointers.cpp.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-04-08 00:30:42 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/14321932238">14321932238</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.4.311"
+TERMUX_PKG_VERSION="1.4.314"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=d784393c9fd2b8c3f9ca71aa14cee5fd0690227e433c16895ea9c6db0e1b43a1
+TERMUX_PKG_SHA256=147e52f6782119170621b71c55e2ff72fc653f088b6ebc99e0ea27f7ebd89d04
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"

